.github/workflows/slow.yaml

```
@@ -223,13 +222,10 @@ jobs:
           run: |
             export MAKE=gmake
             export pjobs="-j`gnproc`"
-            export AUTOMAKE_VERSION=1.16
-            export amver=${AUTOMAKE_VERSION}
+            export AUTOMAKE_VERSION=`ls /usr/local/bin/automake-* | sed 's/.*-//'`
             export ACLOCAL_AUTOMAKE_DIR="/usr/local/share/aclocal-${AUTOMAKE_VERSION}"
             export ACLOCAL_PATH="/usr/local/share/aclocal:/usr/local/share/aclocal-${AUTOMAKE_VERSION}"
```

Do we have to include version-agnostic `/usr/local/share/aclocal`? The rest of this code looks very version-specific... If we do not have to or if you are not sure that we have to, please try to remove it.

```suggestion
            export ACLOCAL_PATH="/usr/local/share/aclocal-${AUTOMAKE_VERSION}"
```